HISTORY.md

```diff
@@ -6,6 +6,10 @@
 ### Behavior Changes
 * Best-efforts recovery ignores CURRENT file completely. If CURRENT file is missing during recovery, best-efforts recovery still proceeds with MANIFEST file(s).
 * In best-efforts recovery, an error that is not Corruption or IOError::kNotFound or IOError::kPathNotFound will be overwritten silently. Fix this by checking all non-ok cases and return early.
+* If `share_table_files` and `share_files_with_checksum` are true, BackupEngine will compare the checksums computed before and after copying of the table files.
```

Comment:
It's not re-reading the file after copying to get the "after copying" checksum, right? It's really comparing checksum before copying to checksum during copying, right?

Reply:
That's right. No re-reading here. It is comparing the checksum before copying to the checksum obtained right after copying is finished.